src/scripts/main.js

```diff
@@ -1,3 +1,15 @@
 'use strict';
 
-// write your code here
+const thead = document.querySelector('thead > tr');
+const posTitleCol = thead.children[1].cloneNode(true);
+
+thead.insertBefore(posTitleCol, thead.lastElementChild);
+
+document.querySelectorAll('tbody > tr').forEach((row) => {
+  const posCol = row.children[1].cloneNode(true);
+
+  row.insertBefore(posCol, row.lastElementChild);
+});
+
+// thead.tr > new col name
+// tbody.tr > new col woith same value as columgn 2
```

Comment:
The comments here contain typos and unclear instructions. 'woith' should be 'with', and 'columgn' should be 'column'. Additionally, the instructions could be clearer about what the intended behavior is for the new column.